api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +179,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+
+	@Test
+	public void valid_shouldRejectNumericReactionValue() throws Exception {
+		Allergy allergy = new Allergy();
+		AllergyReaction reaction = new AllergyReaction();
+		String nonCoded = "sometext";
```

Comment:
Make this more descriptive

Reply:
Ouk.......is it about changing nonCoded to reactionNonCoded